features/Hotels/Tc-22.feature

```diff
@@ -0,0 +1,16 @@
+#TC22
+Feature: SignUp form
+
+    Scenario: Verify error message for invalid data in SignUp form
+        Given I am on hotels landing page
+        When I Click on SignIn link
+        And I Click on SignUp link
+        And I Enter invalid email address with at least '@' symbol 
+        And I Verify error is displayed after invalid email
```

Comment:
Then => And
Any verification should be labelled as Then and following verifications should labelled as And.